Week1/Section3/Exercise7/Source7.cpp

```diff
@@ -47,11 +38,11 @@ int main()
 	{
 		// populate the Lottery array
 		int RandomNumber = 1 + rand() % 20;
-		bool isRandomRepeated = false;
+		bool bIsRandomRepeated = false;
```

Comment:
Hadn't realized this, but why initialize it to false if you are immediately assigning another value to it? Why not write `bool bIsRandomRepeated = IsNumberRepeated(blah blah)`?

Reply:
I didn't see that coming👍